test/1_test_fullpath.js

```diff
@@ -84,14 +87,881 @@ contract('Voucher tests', async (addresses) => {
 
     it('must fail: unauthorized minting ERC-721', async () => {
       await truffleAssert.reverts(
-        contractERC1155ERC721.mint(users.attacker.address, 666),
+        contractERC1155ERC721.mint(users.attacker.address, 666),
+        truffleAssert.ErrorType.REVERT
+      );
+    });
+  });
+
+  describe('Create Voucher Sets (ERC1155)', () => {
+    it('adding one new order / promise', async () => {
+      const txOrder = await contractBosonRouter.requestCreateOrderETHETH(
+        [
+          constants.PROMISE_VALID_FROM,
+          constants.PROMISE_VALID_TO,
+          constants.PROMISE_PRICE1,
+          constants.PROMISE_DEPOSITSE1,
+          constants.PROMISE_DEPOSITBU1,
+          constants.ORDER_QUANTITY1,
+        ],
+        {
+          from: users.seller.address,
+          to: contractCashier.address,
+          value: constants.PROMISE_DEPOSITSE1,
+        }
+      );
+
+      let tokenSupplyKey1;
+
+      truffleAssert.eventEmitted(
+        txOrder,
+        'LogOrderCreated',
+        (ev) => {
+          tokenSupplyKey1 = ev._tokenIdSupply;
+          return (
+            ev._tokenIdSupply.gt(new BN(0)) &&
```

Comment:
since `new BN(0)` && `new BN(1)` are thoroughly used, can we use constants for them?

Reply:
Yes. Good idea. I'll change that.